### ISSUE TITLE
Privileges Configuration shows deleted modules

### ISSUE THREAD

**Issue by joxper** [crocodic-studio/crudbooster#697]
Crudbooster v: 5.4.0.5 : last
Laravel v: 5.4
DB: MariaDB

I added some modules and erase them but they still show on the Privileges Configuration table:

![image](https://user-images.githubusercontent.com/7101877/28831543-1c2cf5d2-76da-11e7-9719-c7e2ee2fd45a.png)

I could help to resolve this issue I just need some guiding @fherryfherry any thoughts?

**Comment by fherryfherry** [comment]
did u add deleted_at field at cms_moduls ?
Pada tanggal 1 Agt 2017 21.55, "joxper" <notifications@github.com> menulis:

**Comment by joxper** [comment]
No, I did not add that field, is the deleted_at field required?

I found two things:

First adding at PrivilegesController on the query to list the moduls:
```
->where([
    ['is_protected',0],
    ['is_active',1],
 ])
 ```

But for all new moduls the is_active is always 0, so I guess this is used for something else?


Second is to delete the entry of the privileges roles table on the the hook_before_delete in the ModulsController.php 

But maybe all is setu for the softdelete?

**Comment by satriaaldy** [comment]
have you resolved this issue? i have the same problem here

**Comment by GhayoorUlHaq** [comment]
Same issue as well
